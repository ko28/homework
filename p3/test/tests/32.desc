Create 2 aliases, delete 1, then run deleted
