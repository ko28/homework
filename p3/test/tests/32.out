alias ls /bin/ls
alias whoami /usr/bin/whoami
alias
ls /bin/ls
whoami /usr/bin/whoami
unalias whoami
alias
ls /bin/ls
whoami
exit
